certified-connectors/absentify/apiDefinition.swagger.json

```diff
@@ -0,0 +1,492 @@
+                },
+                "requester_member": {
+                  "type": "object",
+                  "properties": {
+                    "email": {
+                      "type": "string",
+                      "description": "email"
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "name"
+                    },
+                    "id": {
+                      "type": "string",
+                      "description": "id"
+                    },
+                    "custom_id": {
+                      "type": "string",
+                      "description": "custom_id"
+                    }
+                  },
+                  "description": "requester_member"
+                },
+                "canceld_by_member": {
+                  "type": "object",
+                  "properties": {
+                    "email": {
+                      "type": "string",
+                      "description": "email"
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "name"
+                    },
+                    "id": {
+                      "type": "string",
+                      "description": "id"
+                    },
+                    "custom_id": {
+                      "type": "string",
+                      "description": "custom_id"
+                    }
+                  },
+                  "description": "canceld_by_member"
+                },
+                "declined_by_member": {
+                  "type": "object",
+                  "properties": {
+                    "email": {
+                      "type": "string",
+                      "description": "email"
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "name"
+                    },
+                    "id": {
+                      "type": "string",
+                      "description": "id"
+                    },
+                    "custom_id": {
+                      "type": "string",
+                      "description": "custom_id"
+                    }
+                  },
+                  "description": "declined_by_member"
+                },
+                "leave_type": {
+                  "type": "object",
+                  "properties": {
+                    "id": {
+                      "type": "string",
+                      "description": "id"
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "name"
+                    }
+                  },
+                  "description": "leave_type"
+                }
+              },
+              "description": "details"
+            }
+          }
+        }
+      },
+      "post": {
+        "responses": {},

```

Comment:
@marc-on-github Empty property values can be removed until unless you have a specific reason to keep them.

Reply:
@sriyen-msft thanks for that feedback. Its fixed